src/cli/index.ts

```diff
@@ -77,10 +79,85 @@ $$$$$$\\   $$$$$$$\\  $$\\  $$$$$$\\   $$$$$$$ |$$\\  $$\\  $$\\  $$$$$$\\  $$$$
       open(url.toString());
     });
 
+  program
+    .command("create")
+    .description(
+      "Compile & deploy contracts through your thirdweb dashboard, without dealing with private keys.",
+    ).option("--app",
+      `
+
+        Create a thirdweb app.
+        `,
+    ).option("--ts, --typescript",
+      `
+
+        Initialize as a TypeScript project.
+        `,
+    )
+    .option(
+      "--js, --javascript",
+      `
+
+  Initialize as a JavaScript project.
+`,
+    )
+    .option(
+      "--cra",
+      `
+
+  Initialize as a Create React App project.
+`,
+    )
+    .option(
+      "--next",
+      `
+
+  Initialize as a Next.js project.
+`,
+    )
+    .option(
+      "--vite",
+      `
+
+  Initialize as a Vite project.
+`,
+    )
+    .option(
+      "--use-npm",
+      `
+
+  Explicitly tell the CLI to bootstrap the app using npm
+`,
+    )
+    .option(
+      "--use-pnpm",
+      `
+
+  Explicitly tell the CLI to bootstrap the app using pnpm
```

Comment:
any reason why we need those line breaks here? does it look nicer in terminal or something?